config/routes.rb

```diff
@@ -172,6 +172,7 @@
     end
 
     namespace :sulten, path: "lyche" do
+      get "/" => "lyche#pusser_opp"
```

Comment:
Could you translate this to English? ;-)

Reply:
Done